scripts/common/_configuration.ps1

```diff
@@ -0,0 +1,24 @@
+#
+# Copyright (c) .NET Foundation and contributors. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+#
+
+function loadTestProjectList()
+{
+    return Import-CSV "$RepoRoot\scripts\configuration\testProjects.csv" -Header "ProjectName"
```

Comment:
Not sure I like this Csv approach for listing the projects, though I understand we use them so that we don't have to duplicate it across scripts.

I don't have an alternative solution short of adopting a cross platform build, like Cake of msbuild with CoreCLR.